lib/AppInfo/Application.php

```diff
@@ -169,7 +170,7 @@ public function register(IRegistrationContext $context): void {
 		$context->registerEventListener(DestroyingCircleEvent::class, ListenerFilesDestroyingCircle::class);
 
 		// It seems that AccountManager use deprecated dispatcher, let's use a deprecated listener
```

Comment:
I wonder about this code in general. I didn't find a event `OC\AccountManager::userUpdated` - is this outdated code?